test/backlog/test-data/Altaf Ebrahim t_a Ebrahim & Co v OISC/court_metadata.csv

```diff
@@ -1,2 +1,2 @@
-id,created_datetime,publication_datetime,last_updatedtime,decision_datetime,file_no_1,file_no_2,file_no_3,claimants,respondent,headnote_summary,is_published,main_subcategory_description,sec_subcategory_description,Name,FilePath,Extension,SizeInMB,FileLastEditTime
-5,2002-12-20 00:00:00,2002-12-20 00:00:00,2002-12-20 00:00:00,2002-12-20 00:00:00,2002,10,RTR,Altaf Ebrahim t/a Ebrahim & Co,OISC,,1,Appeals,Refusal to register,Altaf Ebrahim t/a Ebrahim & Co v OISC,JudgmentFiles\j5\2002-010.doc,.doc,0.04,2025-07-18 00:00:00
+id,decision_datetime,file_no_1,file_no_2,file_no_3,claimants,respondent,main_category,main_subcategory,sec_category,sec_subcategory,Name,FilePath,Extension
+5,2002-12-20 00:00:00,2002,10,RTR,Altaf Ebrahim t/a Ebrahim & Co,OISC,Immigration,Appeals,Registration,Refusal to register,Altaf Ebrahim t/a Ebrahim & Co v OISC,JudgmentFiles\j5\2002-010.doc,.doc
```

Comment:
this batch didn't actually have categoreis, just subcategories. Need to figure out how we handle this case before we proceed.